packages/voice/src/VoiceConnection.ts

```diff
@@ -261,10 +261,14 @@ export class VoiceConnection extends TypedEmitter<VoiceConnectionEvents> {
 	 */
 	public set state(newState: VoiceConnectionState) {
 		const oldState = this._state;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 		const oldNetworking: Networking | undefined = Reflect.get(oldState, 'networking');
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 		const newNetworking: Networking | undefined = Reflect.get(newState, 'networking');
```

Comment:
Can't we change the casts to this? Or does ESLint still complain?
```suggestion
		const oldNetworking = Reflect.get(oldState, 'networking') as Networking | undefined;
		const newNetworking = Reflect.get(newState, 'networking') as Networking | undefined;
```

Reply:
If so, then this applies to everywhere disabling this rule.